public/data/praveenscience.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Praveen Kumar Purushothaman",
+  "bio": "Full Stack JavaScript Specialist",
+  "avatar": "https://i.imgur.com/wbUfjzdl.png",
+  "links": [
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/praveentech/",
+      "icon": "briefcase"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/praveenscience",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/praveenscience",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Web Development on my YouTube channel",
+      "url": "https://rb.gy/fupw21",
+      "icon": "youtube"
+    }
+  ],
+  "milestones": []
```

Comment:
Hi @praveenscience , Welcome to the community.
Can you please remove this line? There is no need for this.
```suggestion
```